files-community/Files
# Code Quality: Improved Widget code

## Summary

1st stage of Widgets cleanup.

Since SideBar and Widget use diffierent data models, added SideBar and Widget prefixes so others could understand what a data model is for easily.

Card means Button.
Container means Expander.
FileTagsContainer means a tag container.

- Renamed: **Widget**DriveCardItem
- Renamed: **Widget**FileTagsCardItem
- Renamed: **Widget**FolderCardItem
- Removed: Some of classes that are referenced from nowhere

**Widget models**
![image](https://github.com/files-community/Files/assets/62196528/ecfc5d98-8710-449a-91be-ba3eefd48872)

## Steps To Validate Changes

1. Launch the app
2. Go to Home page
3. See nothing different from what Files stable shows
4. Right click on any widget card item
5. See the context menu shows correctly

## PR Checklist

- [x] **Development**: #4180

## Screenshots

_N/A_

[^1]: If the request hasn't been agreed by the team, this work might be rejected. Make sure that you get approval from the team before opening any PR related the request.
[^2]: If you removed any en-US string resources, make sure that there are no references of those resources. When you add a new en-US string resources, do not make any changes on other languages' resources; [Crowdin](https://crowdin.com/project/files-app) will do that, instead.
